package.json

```diff
@@ -94,6 +94,7 @@
     "@nestjs/schematics": "9.2.0",
     "@nestjs/testing": "9.4.3",
     "@studiohyperdrive/commitlint-config": "^2.2.0",
+    "@types/archiver": "^6.0.2",
```

Comment:
you use here a ^ to define the version. I see there are other dev dependencies that do not have this carrot and use fixed version. Do we have a convention around this or why would we switch between the two?

Reply:
👍 fixed in commit on develop